external-crates/move/move-compiler/src/shared/mod.rs

```diff
@@ -200,13 +237,153 @@ impl CompilationEnv {
         default_config: Option<PackageConfig>,
     ) -> Self {
         visitors.extend([sui_mode::id_leak::IDLeakVerifier.into()]);
+        let filter_attr_name =
+            E::AttributeName_::Known(known_attributes::KnownAttribute::Diagnostic(
+                known_attributes::DiagnosticAttribute::Allow,
+            ));
+        let filter_attributes = BTreeSet::from([filter_attr_name]);
+        let known_filters = BTreeMap::from([
```

Comment:
Could we get the macro (or a macro) back for declaring these? I just find it a hard to parse visually as is.

Reply:
I tried...

Reply:
This looks a lot better, thanks!